sklearn/feature_extraction/text.py

```diff
@@ -1309,6 +1309,26 @@ class TfidfVectorizer(CountVectorizer):
 
     Equivalent to CountVectorizer followed by TfidfTransformer.
 
+    CountVectorizer converts a collection of text documents to a matrix of
+    token counts.
+
+    TfidfTransformer then converts the count matrix from CountVectorizer to a
+    normalized tf-idf representation. Tf is term frequency, and idf is inverse
+    document frequency. This is a common way to calculate the count of a word
+    relative to the appearance of a document.
+
+    The formula that is used to compute the tf-idf of term t is
+    tf-idf(d, t) = tf(t) * idf(d, t), and the idf is computed as
```

Comment:
I think it'll look much nicer if you put the formula in a math block, what do you think?

Reply:
done ✅